egps/egp-5/egp-5.ts

```diff
@@ -39,21 +41,35 @@ async function proposal() {
   // --- main egp logic ---
   /**
    * Steps:
-   * 1. Transfer `amountPastTerms` worth of ELFI to the `aidrop` contract.
-   * 2. Transfer `amountFutureTerms` worth of ELFI to the `futureRewards` contract.
+   * 1. Deploy airdrop contract
+   * 2. Transfer `amountPastTerms` worth of ELFI to the `aidrop` contract.
+   * 3. Transfer `amountFutureTerms` worth of ELFI to the `futureRewards` contract.
    */
 
+  // Deploy the airdrop contract
+  const airdropDeployer = new ethers.ContractFactory(
+    airdropData.abi,
+    airdropData.bytecode,
+    signer
+  );
+  const airdropContract = await airdropDeployer.deploy(
+    signer,
+    hexRoot,
+    "0x5c6D51ecBA4D8E4F20373e3ce96a62342B125D6d", // ELFI Contract address
+    expiration,
```

Comment:
not entirely sure how ethers handles parsing date objects to uint256, maybe change this to `expiration.getTime()` to be safe